sphinx_doc_src/faq.rst

```diff
@@ -14,7 +14,7 @@ Since fish 3.1 you can set an environment variable for just one command using th
     begin; set -lx key value; echo $key; end
 
 How do I run a command every login? What's fish's equivalent to .bashrc or .profile?
-------------------------------------------------------------------------------------
+------------------------------------------------------------------------
```

Comment:
Similarly, this is undoing changes from between the two commits. I'd suggest something like `git reset --soft HEAD^`, then adding *just your changes*, redoing the commit and `git push --force` (yes, this needs force).

Reply:
Sorry, I wasn't paying attention while pulling from the wrong remote.
This is what happens when one is too lazy to make a separate branch.